psh/test-touch.py

```diff
@@ -15,63 +15,32 @@
 import psh.tools.psh as psh
 import trunner.config as config
 from psh.tools.common import (CHARS, assert_present, assert_file_created, assert_dir_created,
-                              assert_random_files, get_rand_strings, create_testdir)
+                              assert_random_files, get_rand_strings, create_testdir, assert_mtime_change)
 
 ROOT_TEST_DIR = 'test_touch_dir'
 
 
-def assert_timestamp_change(p, touched_files, uptimes, dir=''):
-    files = psh.ls(p, dir)
-    for file in files:
-        if file.name in touched_files:
-            uptime = uptimes[file.name]
-            # uptime has to be got before touch
-            # to prevent failed assertion when typing uptime in 00:59 and touch in 01:00
-            # 1 min margin is applied
-            next_minute = f'{(int(uptime.minute)+1):02d}'
-            if file.date.time == f'{uptime.hour}:{next_minute}':
-                uptime_str = f'{uptime.hour}:{next_minute}'
-            else:
-                uptime_str = f'{uptime.hour}:{uptime.minute}'
-            # when writing test, system date is always set to Jan 01 00:00
-            # so all touched files should have the following timestamp: Jan 01 00:00 + uptime
-            msg = f"The file's timestamp hasn't been changed after touch: {dir}/{file.name}, timestamp = {file.date}"
-            assert file.date == ('Jan', '1', uptime_str), msg
-
-
-def assert_symlinks(p):
-    uptimes = {}
-    # all psh commands are symlinks
-    psh_cmds = psh.get_commands(p)
-    for psh_cmd in psh_cmds:
-        uptimes[psh_cmd] = psh.uptime(p)
-        msg = f"Prompt hasn't been seen after the symlink touch: /bin/{psh_cmd}"
-        psh.assert_cmd(p, f'touch /bin/{psh_cmd}', '', msg)
-
-    assert_timestamp_change(p, psh_cmds, uptimes, '/bin')
-
-
 def assert_executable(p):
     # psh is the example of executable which exists in file system
     msg = "Prompt hasn't been seen after the executable touch: /usr/bin/hello"
-    uptime = psh.uptime(p)
+    date = psh.date(p)
     if config.CURRENT_TARGET in config.SYSEXEC_TARGETS:
         psh.assert_cmd(p, 'touch /syspage/psh', '', msg)
```

Comment:
```suggestion
        psh.assert_cmd(p, 'touch /syspage/psh', msg=msg)
```
Don't pass default arguments, use keywords.